apps/web/src/hooks/useApproveCallback.ts

```diff
@@ -8,12 +8,13 @@ import { useHasPendingApproval, useTransactionAdder } from 'state/transactions/h
 import { calculateGasMargin } from 'utils'
 import { getViemErrorMessage } from 'utils/errors'
 import { isUserRejected, logError } from 'utils/sentry'
-import { Address } from 'viem'
+import { Address, Hex, SendTransactionReturnType, encodeFunctionData, parseAbi } from 'viem'
 import { useAccount } from 'wagmi'
-import useGelatoLimitOrdersLib from './limitOrders/useGelatoLimitOrdersLib'
+import { usePaymaster } from 'views/Swap/V3Swap/components/Paymaster/hooks/usePaymaster'
```

Comment:
the dependency relationship should be the other way around. We can import `src/hooks` in `views`, but we shouldn't import `views` in `src/hooks`.

If we're importing `views` inside a common hook, it means that the imported module should also be a top-level common module